R/TealAppDriver.R

```diff
@@ -253,13 +253,14 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
     get_active_module_tws_output = function(tws, which = 1) {
       checkmate::check_number(which, lower = 1)
       checkmate::check_string(tws)
-      table <-
-        rvest::html_table(
-          self$get_html_rvest(
-            self$active_module_element(sprintf("%s-table-with-settings", tws))
-          ),
-          fill = TRUE
-        )
+      table_selector <- ifelse(
```

Comment:
what if we have both table in module ?

Reply:
I did not see cases like this in all the 36 modules in tmc.

Reply:
But, I see your point, I will make this as an argument to make the selection.

Reply:
Done in b36be57b69269c7703ed2843ce7cc84eba087996